test/gl-common.hpp

```diff
@@ -3,7 +3,7 @@
 
 #include <GL/glew.h>
 #include <GL/gl.h>
-#include <GL/glext.h>
+// #include <GL/glext.h>
```

Comment:
Is this no longer needed? Why not delete if so?

Reply:
glext.h and glew.h are not compatible. It cannot compile in my computer.

https://stackoverflow.com/questions/65009898/incompatible-function-prototypes-between-glew-h-and-glext-h

I will delete it.